src/Ops.Controllers/Areas/SiteManagement/SegmentsController.cs

```diff
@@ -274,6 +284,19 @@ var pageLayoutId
                         "This segment is used for multiple locations: {0}",
                         string.Join(", ", locations.Select(_ => _.Name)));
                 }
+
+                var episode = await _podcastService.GetEpisodeBySegmentIdAsync(segment.Id);
```

Comment:
A permission check for podcasts permission needs to be added to the `HasSegmentPermissionAsync` function in the SegmentsController, currently a user with just podcast permissions won't be able to view or edit the segment details.

Reply:
Resolved in commit 577d9e3